docs/source/teams/roles_and_permissions.rst

```diff
@@ -105,14 +139,28 @@ Admins and users with the **Can manage** permission on a dataset can configure
 a dataset's permissions under the dataset's
 :ref:`Manage tab <teams-managing-datasets>` in the FiftyOne Teams App.
 
-A user's permissions on a dataset is the maximum of their permissions from the
+In FiftyOne Teams, dataset permissions for a user are determined by both the 
+access they receive from their group's permissions and individual permissions 
+assigned to them. This tiered permissions system is 
+designed to facilitate management at scale. 
+
+A user’s permissions on a dataset is the maximum of their permissions from the 
 following sources:
 
 -  Admins implicitly have full access to all datasets
 -  Members have the dataset's
    :ref:`default access level <teams-default-access>`
 -  Users may be granted :ref:`specific access <teams-specific-access>` to the
    dataset
+-  Users may be members of one or more groups, each of which may have
+   :ref:`specific access <teams-specific-access>` to the dataset
+
+.. note::
+
+   User role determines the highest level of access that a user can be granted 
+   to a dataset. For example, a user with guest role can be added to a group 
+   with **Can edit** permission to a dataset, but the guest will have
+   **Can view** permission instead of **Can edit** permission of the dataset.
```

Comment:
suggestion: add:

```
, because Guests may only have **Can view** permissions to datasets.
```

to the end of this sentence to make the example very concrete.